x/emissions/keeper/queryserver/query_server_inferences.go

```diff
@@ -14,7 +15,11 @@ import (
 
 // GetWorkerLatestInferenceByTopicId handles the query for the latest inference by a specific worker for a given topic.
 func (qs queryServer) GetWorkerLatestInferenceByTopicId(ctx context.Context, req *emissionstypes.GetWorkerLatestInferenceByTopicIdRequest) (_ *emissionstypes.GetWorkerLatestInferenceByTopicIdResponse, err error) {
-	defer metrics.RecordMetrics("GetWorkerLatestInferenceByTopicId", time.Now(), &err)
+	labels := map[string]string{
+		"worker_address": req.WorkerAddress,
+		"topic_id":       strconv.FormatUint(req.TopicId, 10),
+	}
+	defer metrics.RecordMetrics("GetWorkerLatestInferenceByTopicId", time.Now(), &err, labels)
```

Comment:
Is it needed to have labels on queries? I understand the need of labels when it's related to a mutation in the state but I'm not sure it's relevant for queries